src/Sulu/Bundle/AdminBundle/Resources/js/utils/TestHelper/fieldTypeDefaultProps.js

```diff
@@ -13,6 +13,8 @@ const fieldTypeDefaultProps = {
     onChange: jest.fn(),
     // $FlowFixMe
     onFinish: jest.fn(),
+    // $FlowFixMe
+    onSuccess: jest.fn(),
```

Comment:
We try to use the most default value in here, so this should be `undefined`. If something is not working because of that, it would need separate fixing. (Would also save the `$FlowFixMe` comment above)